app-devel/mise/autobuild/templates

```diff
@@ -0,0 +1,8 @@
+Template: mise/activation_hints
+Type: note
+Description: Activate mise for Your Shell
+ mise is installed yet to be manually activated for your shell. Please refer to
```

Comment:
mise is now installed, but you would need to configure it for your shell. Please see:
.
https://mise.jdx.dev/getting-started.html#activate-mise
.
For detailed instructions.